Change url for geolocation

tacigomess opened the pull request:
Geolocation website has change the domain due trademark, now is
geolocation-db.com/json the endpoint.

tacigomess committed:
Updating weather-api for python3

The weather-api name has change in python3 to python3-weather-api.

tacigomess committed:
Change url for geolocation

Geolocation website has change the domain due trademark, now is
geolocation-db.com/json the endpoint.

tacigomess committed:
Change url for geolocation

Geolocation website has change the domain due trademark, now is
geolocation-db.com/json the endpoint.

satyammittal commented on google_places.py:
```diff
@@ -40,7 +40,7 @@ def get_location():
     # u'latitude': 17.3753, u'state': u'Telangana', u'IPv4': u'157.48.48.45',
     #  u'country_code': u'IN', u'country_name': u'India', u'postal': u'500025'}
-    url = "https://geoip-db.com/json"
+    url = "https://geolocation-db.com/json"
```
👍 figuring out. It would be great if we could add this in config file.

satyammittal committed:
Merge pull request #20 from tacigomess/fix-weather-api

Updated requirement for python3 support.

tacigomess committed:
Fix geolocation

This patch adds the geolocation url in config.py and set it
properly to the new geolocation website

tacigomess committed:
Fix geolocation

This patch adds the geolocation url in config.py and set it
properly to the new geolocation website

pep8speaks commented:
Hello @tacigomess! Thanks for updating this PR. 









There are currently no PEP 8 issues detected in this Pull Request. Cheers! :beers: 

##### Comment last updated at 2020-03-09 21:23:50 UTC

tacigomess committed:
Fix comments

tacigomess committed:
Fix comments

tacigomess committed:
raw_input was replaced to input()

Python3 use input() instead raw_input()

arxcruz committed:
Enable tox support

This patch enable tox support on chatbot project.
Initially, it's skipping several pep8 otherwise it won't pass the CI.
Once the patch get merged, it's possible to start remove the ignored
peps.
This will allow tests to be executed locally, before the patch get
submited, and improve the code quality.
This also add the test-requirements.txt and change the travis yaml file
to use it, instead of the pip install commands.

satyammittal committed:
Merge pull request #23 from tacigomess/fix-raw_input

raw_input was replaced to input()

satyammittal commented on google_places.py:
```diff
@@ -40,8 +41,8 @@ def get_location():
-    url = "https://geolocation-db.com/json"
-    response = urllib.urlopen(url)
+    url = config.geolocation_api
+    response = urllib.request.urlopen(url) #outra fix-urllib.urlopen
```
You have already created separate merge request for this. So you can exclude this :).

satyammittal committed:
Merge pull request #24 from arxcruz/enable-tox

Enable tox support

tacigomess committed:
Merge branch 'fix-geolocation' of https://github.com/tacigomess/voice-enabled-chatbot into fix-geolocation

satyammittal commented:
Hi @tacigomess , you can update your branch by first fetching changes from master and then rebase.

Example:

```
git fetch
git rebase origin/my-branch
```
